### ISSUE TITLE
Feature Suggestion

### ISSUE THREAD

**Issue by anks999999** [FarbstoffRSL/RSL-Helper#159]
Firstly, I am very appreciate and love your work. It help me too much, as a newbie.

i just suggest a basic function seem hard to do but very needed : auto upgrade rank food. the idea just select a number of max level fooder, then find a number of level  1 same rank same quality to upgrade to next rank
example : you have 10 level 10 rank 1 common, select 5, then it will find 5 and upgrade it to rank 2
it will very useful for new player progressing like me

finally, thank you very much for your RSL
Anks

**Comment by Dwza** [comment]
i dont think that this behavior would be allowed by plarium. People should still play the game.

**Comment by Arthvendis** [comment]
We already have things that basically mimic what Plarium is often trying to sell (Multi Level XP vs Multi-battles) - if that's ok, then why would auto rank-up be a dealbreaker? Not trying to be cheeky, just genuinely asking.
